### ISSUE TITLE
translation file uploads do not clear "needs editing" strings if upload has them emptied

### ISSUE THREAD

**Issue by yilmazdurmaz** [WeblateOrg/weblate#11596]
### Describe the issue

I am working on Godot translations and I am trying to clear automatic translations as they are mostly wrong (since they are copied from a previous work by devs)

The total is already over 10k, and "needs editing" strings are quite an attention killer for me. So, when an update comes in, I usually clear them all before continuing translating.

Since Weblate does not have such an editing tools (or I could not discover yet?), I download the po file, batch clear fuzzy strings with regex, and upload the file back with these options:
- Add as translation
- Import as translated
- Change translated and approved strings

I might have forgotten how things worked since I had my last batch many months ago, but I remember these option would replace everything on Weblate with my uploaded file.

Yet, today I am having difficulty with those nasty strings. The uploads comes with this message, saying no change applied on them:
> Processed 24568 strings from the uploaded files (skipped: 4666, not found: 0, updated: 0).

For the moment, I replaced them all with a single space character, `msgstr " "`, as it is better to have nothing than having wrong information.

Please, correct this issue. I know this is dangerous action but those 3 options are meant for this (let alone along the uploading itself).

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar filed issues in this repository.

### Steps to reproduce the behavior

1- have a project with "needs editing" strings with po style content
2- download po file
3- change translation portions as `msgstr ""`and remove `fuzzy` line (makes it untranslated)
4- save and upload with options in the main text above.

### Expected behavior

all "needs editing" should be removed and there should be only "untranslated/unfinished" shown

### How do you run Weblate?

weblate.org service

**Comment by yilmazdurmaz** [comment]
> What you can do is to use _Replace existing translation file_ instead

There is no "replace" option in the upload page. at least not for us contributor translators.

**Comment by nijel** [comment]
Yes, you need more permission for that (because it can remove strings as well).

**Comment by yilmazdurmaz** [comment]
@nijel, sorry for slow responses, health issues in the family.

this permission requirement takes us back to square one; that for us translators, these options triplet should replace/clear translated-but-needs-editing strings without admin level permissions.

For these option triplet, you may add an extra checkbox with a big warning that if process continues, for example, it may damage the source, and ask the user to double or triple check what they did.

I feel the urge to repeat my original attention in a little bit different way:

When a project's maintainers/admin make changes on translations through scripting, usually po files, they copy old translations into new strings with zero history, and remove old ones with their history. 

Somehow on the way, a lot of these translation matches 0% to the source, and thus the truth level drops sharp on the total.

There are no other actively participating translator other than me in the last 6 months, and I was very disfocused aboutt he project. so these needs-editing translations were accumulated over 2000 strings. This loads a huge strain on my nerves as my best was 100-200 per day, and i do not have regular access recently. clearing them there and then is the best way to ease the strain.

another stress is that since this is a po type project, they will use whatever translated string is present without ever proof reading (which is what i do basically) and thus i want to clear all those strings. when no translation is present, they use source strings instead, and it is best to have no translation ever instead of having wrong translations.

you may say why not to ask maintainers for more project permission. but as you may understand, open source projects may not see them around for long, so even if i get access, then the next person i trust the flag may not get such permission in time. 

Anyways, I really appreciate you taking your time and bear with us. Thank you very much. If you got more time, I would like to hear more from you about how else could we get a solution. For the moment I am a bit burnt out, so until next time.

**Comment by nijel** [comment]
Having an additional option for blank strings similar to what we do for fuzzy is the way to go:

https://github.com/WeblateOrg/weblate/blob/e7f86b3b35cdc14546930d368b340dd215cdb836/weblate/trans/forms.py#L670-L678

The default behavior would be to skip these, but it could be changed.

**Comment by yilmazdurmaz** [comment]
That would be nice to implement when you have time. It is not something urgent after all.

By the way, you have currently a nice feature that if someone empties an existing string it is marked as "previously translated". My last edits of about 1600 strings are now like that, and it feels much better this way.

The two would also work very nicely together. If a newbie member makes mistakes, we have a way to detect.
